spec/services/users/profile_spec.rb

```diff
@@ -8,15 +8,35 @@
   let!(:in_progress_form_user_uuid) { create(:in_progress_form, user_uuid: user.uuid) }
   let!(:in_progress_form_user_account) { create(:in_progress_form, user_account: user.user_account) }
 
+  let(:cassettes_good_vet_status) do
```

Comment:
Same comment here

Reply:
This is not a cassette name, it is a variable name. The cassettes are in line 13 and 14.